### DISCUSSION TITLE
Problem with postcss.config

### DISCUSSION THREAD

**Discussion by Legoog** [mantinedev/mantine#4840] [Feedback]
I just begun to migrate from version 6 to version 7. I use the vite-template. By running the Welcome example I get the following error: _`A PostCSS plugin did not pass the `from` option to `postcss.parse`. This may cause imported assets to be incorrectly transformed. If you've recently added a PostCSS plugin that raised this warning, please contact the package author to fix the issue.`_

**Comment by mattkrins** [comment]
I am also using Vite and getting this error.
~~Everything seems to working fine still and styles are getting applied, so it is most likely a warning, but still very annoying.~~
**[edit]** Actually no, styles are not working. The basic css seems to get applied, but it fails with vars and such. 
Do we need to open a issue with Vite?

**Reply by rtivital** [reply]
You can use vite template, styles are working correctly there

**Reply by Legoog** [reply]
In my case, using one of the vite templates, the styles are not working when using vars, rem() ect.

**Reply by mattkrins** [reply]
> You can use vite template, styles are working correctly there

@rtivital Vite template is not working correctly either.
The template does not include any theming outside of Mantine itself, which appears to be the problem.

~~Using the template works, but as soon as you try to apply styling using ``import classes from './whatever.module.css';`` the PostCSS error appears.~~
**[Edit]** Using the template works, but as soon as you try to apply styling using ``import classes from './whatever.module.css';``, if you use the light-dark function in the css module, the PostCSS error appears.

**Comment by mattkrins** [comment]
I have narrowed it down to the light-dark postcss-preset-mantine function.
Opened an issue here:
https://github.com/mantinedev/postcss-preset-mantine/issues/7

**Reply by Legoog** [reply]
Yes. It seems to be just that function. Thanks a lot!
